docs/section12/Rule12-7.md

```diff
@@ -0,0 +1,38 @@
+
+# Receptacle - Rule 12-7
+
+**Rule ID:** 12-7  
+**Rule Description:** User RMR Receptacle Power in Baseline RMR?  
+**Rule Assertion:** Baseline RMR = User RMR  
+**Appendix G Section:** Receptacle  
```

Comment:
Put the specific table number on this and other receptacle rules

Reply:
Updated in #35